projects/aca-content/assets/app.extensions.json

```diff
@@ -1177,6 +1177,23 @@
               }
             }
           ]
+        },
+        {
+          "id": "app.viewer.separator.1",
+          "type": "separator",
+          "order": 11000
+        },
+        {
+          "id": "app.viewer.close",
```

Comment:
I don't think we should add new button for that, the close button already exist in ADF and ACA should just set the position value which should be passed to the ADF component which will correctly place the button.

Reply:
@MichalKinas to move the close button to the right side in the header we have to add the close button in toolbar-actions, and for left position we are using the existing button only.

Reply:
Why do you think you have to add new button in ACA? This is another thing that has to be maintained, is not documented properly, has naming issues and rule `canClosePreview` doesn't do what it's name suggests. This should be part of ADF component which should take a param with position of close button. This solution works in ACA only and overrides default component behaviour, I still think that the solution should be adding new param for ADF component and component itself should decide how to display the close button

Reply:
@MichalKinas as per your suggestion I have removed the close button from toolbar-actions,now I am planning to add one more button in ADF viewer.component.html which will be show/hide as per the position set by customer in app.config.json which can be configuration from ACA or ADF.